### PR TITLE
atlassian-jira 6.3.12 (new formula)

### PR DESCRIPTION
**Pull request by marsmensch**
atlassian-jira 6.3.12 (new formula)

### CODE REVIEW

File: Library/Formula/atlassian-jira.rb [hunk 1]
```diff
@@ -0,0 +1,64 @@
+# atlassian-jira 6.3.12 (new formula)
```
Comment (DomT4) [inline Library/Formula/atlassian-jira.rb]: You can kill this line :wink:

File: Library/Formula/atlassian-jira.rb [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+
+class AtlassianJira < Formula
+  homepage "https://www.atlassian.com/software/jira/"
+  url "http://www.atlassian.com/software/jira/downloads/binary/atlassian-jira-6.3.12.tar.gz"
+  sha1 "3638292cd9b280d8aee266b5639a6cc8da4de866"
+
+  depends_on :java => "1.7"
+
+  def install
+    # TODO
```
Comment (DomT4) [inline Library/Formula/atlassian-jira.rb]: And this one :grinning:

File: Library/Formula/atlassian-jira.rb [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+  plist_options :manual => "#{HOMEBREW_PREFIX}/opt/atlassian-jira/libexec/bin/start-jira.sh"
+
+  def plist; <<-EOS.undent
+    <?xml version="1.0" encoding="UTF-8"?>
+    <!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+    <plist version="1.0">
+      <dict>
+        <key>Label</key>
+        <string>#{plist_name}</string>
+
```
Comment (DomT4) [inline Library/Formula/atlassian-jira.rb]: Personally hate whitespaces in ` plists ` but I'll leave that call up to the maintainers :)

**Comment by mikemcquaid** [comment]
Again, what's the license for Jira?

**Comment by marsmensch** [comment]
Thanks for your feedback! I will work on these tomorrow after the first coffee. Do i need to create a new pull request afterwards, or will this one update?

**Comment by marsmensch** [comment]
@mikemcquaid proprietary (mostly), just as confluence and bamboo https://www.atlassian.com/licensing/jira

**Comment by mikemcquaid** [comment]
I think these Atlassian projects probably aren't suitable if they require a separate license flow and aren't e.g. usable for commercial purposes.

**Comment by adamv** [comment]
Possible candidate for https://github.com/atlassian/homebrew-tap ?

**Comment by marsmensch** [comment]
@jacknagel @mikemcquaid what do you mean with "license flow"? I have no intent to put them into the atlassian homebrew-tap but if you don't want these Formulas, i will probably re-create them as Casks.

**Comment by mikemcquaid** [comment]
@marsmensch Well, if you're using it for non-OSS/edu purposes you need to acquire a license. That's unlike most of the software we have in here. For what it's worth I don't think Cask magically solves that problem either.

**Pull request closed without merging**